packages/trpc/server/routers/viewer/dsync/_router.tsx

```diff
@@ -0,0 +1,66 @@
+import authedOrgAdminProcedure from "../../../procedures/authedProcedure";
+import { router } from "../../../trpc";
+import { ZCreateInputSchema } from "./create.schema";
+import { ZDeleteInputSchema } from "./delete.schema";
+import { ZGetInputSchema } from "./get.schema";
+
+type DsyncRouterHandlerCache = {
+  create?: typeof import("./create.handler").createHandler;
+  get?: typeof import("./get.handler").getHandler;
+  delete?: typeof import("./delete.handler").deleteHandler;
+};
+
+const UNSTABLE_HANDLER_CACHE: DsyncRouterHandlerCache = {};
+
+export const dsyncRouter = router({
```

Comment:
Can we use the current desired boilerplate as done here? https://github.com/calcom/cal.com/pull/10849/files

Reply:
Dsync routes shouldn't be public. I noticed that PR only made changes to the public router. I looked at the event type `_router` file and it's following the same convention.